src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/category/CategoryPageObject.java

```diff
@@ -28,6 +28,7 @@ public String getCategoryName() {
 
     public void verifyCategoryPageTitle(String title) {
         wait.forElementVisible(categoryHeader);
+        System.out.println("Category Header: " + categoryHeader.getText());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)